core/src/main/web/app/helpers/core.js

```diff
@@ -75,15 +75,32 @@
       newStrategy.manualEntry = function() {
         newStrategy.manualName = this.input ? this.input.split('/').pop() : "";
       };
-      newStrategy.checkCallback = function(result){
-        if (result && result.indexOf('.bkr') === -1){
-          $rootScope.$broadcast("SELECT_DIR",{
-            find_in_home_dir: true,
-            path: result
+      newStrategy.checkCallback = function (result) {
+        var deferred = bkUtils.newDeferred();
+        if (!result){
+          deferred.resolve({
+            result: true
           });
-          return false;
+        }else {
+          bkHelper.httpGet(bkHelper.serverUrl("beaker/rest/file-io/isDirectory"),
```

Comment:
we can't make a rest call directly here.  the file dialog is created using a strategy object to access the filesystem.
this strategy object might access a different filesystem than the normal unix one on this server.